enforcer-rules/src/main/java/com/google/cloud/tools/opensource/enforcer/LinkageCheckerRule.java

```diff
@@ -146,8 +173,12 @@ public void execute(@Nonnull EnforcerRuleHelper helper) throws EnforcerRuleExcep
           new DefaultDependencyResolutionRequest(mavenProject, session);
       DependencyResolutionResult dependencyResolutionResult =
           projectDependenciesResolver.resolve(dependencyResolutionRequest);
-      return dependencyResolutionResult.getDependencies().stream()
```

Comment:
`dependencyResolutionResult.getDependencies()` was not returning items in breadth-first order.

Reply:
I suspect this should be fixed first in an independent PR.

Reply:
Sure. Let me create one.